app/serializers/submission_serializer/sections/capital_details.rb

```diff
@@ -1,9 +1,9 @@
 module SubmissionSerializer
   module Sections
     class CapitalDetails < Sections::BaseSection
-      def to_builder # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Metrics/CyclomaticComplexity
+      def to_builder # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
         Jbuilder.new do |json| # rubocop:disable Metrics/BlockLength
-          next unless capital && requires_full_means_assessment?
+          next unless capital && extent_of_means_assessment_determined? && requires_full_means_assessment?
```

Comment:
As of this PR we're serialising the application earlier in its lifecycle. This guard is required to hide this section when the extent of means is not yet known when the upload evidence page is visited.